src/resolvers-cassandra/Settings/index.js

```diff
@@ -4,7 +4,8 @@ const mutations = require('./mutations');
 const queries = require('./queries');
 
 module.exports = {
-  createOrReplaceSite: mutations.createOrReplaceSite,
+  createSite: mutations.createSite,
+  removeSite: mutations.removeSite,
```

Comment:
This may not be a big deal, but I thought I'd call it out: this is a non-backwards compatible change from the point of view of the non-cassandra resolver. Should we just add the two new ones and have the `createOrReplaceSite` always call `reject`? Also, should we add the `createSite` and `removeSite` to the old non-cassandra side?